OpenRA.Game/Game.cs

```diff
@@ -69,6 +69,15 @@ static void JoinInner(OrderManager om)
 			ConnectionStateChanged(OrderManager);
 		}
 
+		public static void ChangeRes(Size size)
```

Comment:
This isn't going to work in the long term: you can't know ahead of time everything that needs to respond to resolution changes.

You should instead set up an `OnResolutionChange` event in the Renderer class that everything else attaches callbacks to.